src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -102,7 +107,8 @@ internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
                 var returnTypeName = ztSearchElement.Descriptor.ReturnType.Name;
 
                 var descriptor = ztSearchElement.Descriptor;
-                if ((returnTypeName == inputPortType)
+                if (descriptor.ReturnType.ToString() == inputPortType)
```

Comment:
can we use `{` around if statements - it's in our coding style, if we're going to stop doing this let's update the style guide.
https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards#conditional-blocks